### PR TITLE
Add tests for LocationHelper

### PR DESCRIPTION
**Pull request by ekigamba**

### CODE REVIEW

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/location/helper/LocationHelperTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5549852&bid=18197227)

**Comment by ekigamba** [comment]
To fix commented out tests in https://github.com/OpenSRP/opensrp-client-core/issues/553

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=1.11.8-SNAPSHOT
+VERSION_NAME=1.11.8.1-KIG-SNAPSHOT
```
Comment (allan-allay) [inline gradle.properties:1]: Should this be renamed?

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- opensrp-app/src/test/java/org/smartregister/location/helper/LocationHelperTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-client-core/pullRequest?prid=5549852&bid=18197227)

File: gradle.properties (hunk 1 above)
Comment (ekigamba) [reply gradle.properties]: Yea, this has been reverted back to what was there. There is no need to change since it only contains tests

**Pull request merged**